src/Pages/HomeScreen.tsx

```diff
@@ -1,8 +1,49 @@
 import * as React from "react";
-import { Button, Text, View } from "react-native";
-import { HomeScreenProps } from "../Navigation/NavigationTypes";
+import { Button, Image, Text, View } from "react-native";
+import { HomeScreenProps } from "../navigation/navigationTypes";
+import { useDispatch, useSelector } from "react-redux";
+import { Category, Meal } from "../types/types";
+import { fetchCategories } from "../redux/actions/categoriesActions";
+import { fetchRandomMeal } from "../redux/actions/mealsActions";
+import { searchMeal } from "../redux/actions/searchActions";
 
-export default function HomeScreen({ navigation }: HomeScreenProps) {
+const HomeScreen = ({ navigation }: HomeScreenProps) => {
+  const dispatch = useDispatch();
+  const categories = useSelector((state: any) => state.categories);
+  const randomMeal = useSelector((state: any) => state.randomMeal);
+  const searchMealResults = useSelector(
+    (state: any) => state.searchMealResults
+  );
+
+  React.useEffect(() => {
+    dispatch(fetchCategories());
+    dispatch(fetchRandomMeal());
+    dispatch(searchMeal("Arrabiata"));
+  }, []);
+  console.log(
```

Comment:
do we need this?

Reply:
Yes, I think that logs are important for testing api requests

Reply:
I think that we can keep all logs for this commit, as we don't have a user interface yet and I will delete all of them after the UI is ready. Do you agree?